README.md

```diff
@@ -79,18 +100,32 @@ The example above will return 3 chunks (header prefix not shown):
 ### Unchunking
 
 This library works both if chunks are sent in ordered or unordered
-manner. Because ordering is not guaranteed, the Unchunker instance
+manner. Because ordering is not guaranteed, the `Unchunker` instance
 accepts chunks and stores them in an internal data structure. As soon as
 all chunks of a message have arrived, a listener will be notified.
 Repeated chunks with the same serial will be ignored.
 
-Create the Unchunker instance:
+Create an instance for one of the `Unchunker` classes:
 
 ```javascript
-let unchunker = new Unchunker();
+let unchunker = new BlobUnchunker();
+```
+
+or
+
+```javascript
+let unchunker = new Uint8ArrayUnchunker();
+```
+
+Then, register a message listener. For the `BlobUnchunker`:
+
+```javascript
+unchunker.onMessage = (message: Blob, context: any[]) => {
+    // Do something with the received message
+};
```

Comment:
Hm, I just noticed that this is TypeScript syntax, not JavaScript. Not sure if we should reword something...